### ISSUE TITLE
1.9 Kicks randomly Players | New

### ISSUE THREAD

**Issue by MultiDJRoni** [MylesIsCool/ViaVersion#92]
### What version of ViaVersion are you using? Type /ver viaversion
0.4.6

### What version of Spigot are you using? Type /ver
-> http://prntscr.com/aapage

### What plugins are you using? /plugins
http://prntscr.com/aapani

### How does this error happen? login? using an item? 
If i place a Block i will get kicked with this Message: http://prnt.sc/aap7ut
I also get randomly kicked when im Walking arround.

### Is there an error in the console? Use pastebin.com. Is there a kick message?
https://paste.ee/p/p1gjq#

**Comment by MultiDJRoni** [comment]
Another Bug:

If i Block with a Sword in 1.9, in 1.8 im blocking. But if i unblock in 1.9, in 1.8 im Still blocking with my Sword and it looks like im having NoSlowDown.

Could you fix this by do it like HyPixel, with a Shield but in 1.8 your blocking, and if you unblock it removes the Shield und unblocks in 1.8.

Ps. Nice Plugin :)

**Comment by Paulomart** [comment]
We currently focusing on fixing bugs. Gameplay stuff comes right after that :)

**Comment by HAlexTM** [comment]
You should open another issue for this, anyway you have a lot of unloaded plugins (crashes) and a lot of plugins that interfere with ViaVersion

are you sure of the version? (0.4.8) as the latest is 0.4.6...

**Comment by MylesIsCool** [comment]
I'd suggest removing plugins and testing like the ImageFireworks it looks already you have a lot of issues.

**Comment by MultiDJRoni** [comment]
Oh sorry, here are the right logs: https://paste.ee/p/1xL0I#
I took the logs from the Lobby Server :D 

The Version ist 0.4.6, also my fail.

**Comment by HAlexTM** [comment]
What is ChunkLoader? can it cause the kick?

**Comment by MultiDJRoni** [comment]
ChunkLoader is a Plugin that keeps the selected Chunk loaded. /chunkloader set and this Chunk will being always loaded.

**Comment by MylesIsCool** [comment]
Could you update to latest version and test this still happens?

**Comment by MultiDJRoni** [comment]
Yes, i will try it :)

**Comment by MultiDJRoni** [comment]
The BlockBug is fexed, but there are some new Bugs.

The Chunk Loading is Bugy (Its not the ChunkLoader Plugin, i tested it) and i sometimes cant playe Blocks until i die, usually its also becaused of the Chunk Bug.

The Version, Plugins, Bugs etc. you can see in the Video!
Link: https://youtu.be/J5WxUA1ZvAw

**Comment by MylesIsCool** [comment]
Please open a new issue for the placing blocks if it's different. Otherwise, #38 has been reported.
